### PR TITLE
Implement add_suffix, add_prefix

### PR DESCRIPTION
**Pull request by phofl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ API Coverage
 - `partitions`
 - `prod`
 - `rename`
+- `add_prefix`
+- `add_sufix`
```
Comment (rjzamora) [inline README.md:104]: Could we keep this list in alphabetical order for now? :)

**Review by rjzamora — COMMENTED** [review]
Just a suggestion for the `README`. Otherwise, looks good.

**Optional**: Doesn't need to be in this PR, but we probably want `_simplify_up` logic that is similar to `RenameFrame`. For both these cases, we could probably also push down filters as well.

File: README.md (hunk 1 above)
Comment (phofl) [reply README.md]: Changed

**Comment by phofl** [comment]
Added simplify steps as well

I'll have to look into filters more broadly for Blockwise stuff. So will keep this out of here

**Review by rjzamora — APPROVED** [review]
Nice. Thanks!

**Pull request merged**